### DISCUSSION TITLE
MariaDB

### DISCUSSION THREAD

**Discussion by EWB24** [leokhoa/laragon#735] [General]
Hello,

When selecting the Maria DB Database you get 2 error messages.

![Capture d'écran 2024-03-24 160450](https://github.com/leokhoa/laragon/assets/59571651/03fa99f7-b6d1-4c28-95ad-d79a2ed40623)
![Capture d'écran 2024-03-24 160617](https://github.com/leokhoa/laragon/assets/59571651/37aec61f-529b-4dbf-ba37-0a8a2dacc7f7)

The file mysqld.log is empty

In the frontend :
The server returned a "500 - Whoops

What am I doing wrong ?

**Comment by irev** [comment]
i have the same problem
![image](https://github.com/leokhoa/laragon/assets/1285689/2dfa6242-d467-46ae-a7e5-e08240b66577)
